docs/design/coreclr/jit/hot-cold-splitting.md

```diff
@@ -0,0 +1,173 @@
+# Hot/Cold Code Splitting
+
+This document describes the current state of hot/cold splitting in the JIT.
+
+Hot/Cold splitting is a PGO optimization where generated code is split into frequently-executed ("hot") and
```

Comment:
Hot/cold splitting is not strictly Profile Guided Optimization. It is quite effective even without profile data.